config.json

```diff
@@ -6,16 +6,36 @@
   },
   "server": {
     "documentRoot": "data",
-    "httpPort": 4544,
-    "httpsPort": 4545,
-    "enableSsl": true,
+    "httpPort": 4545,
+    "httpsPort": 4546,
+    "enableSsl": false,
     "keystorePath": "keystore.jks",
-    "keystorePassword": "password"
+    "keystorePass": ""
```

Comment:
To me, this seems to be an odd change. I would suggest to keep a) the key as `keystorePassword` and b) to also keep the **default developement** password in the config. Upon deployment this needs to be set context-aware anyways